controllers/openstackvmset_controller.go

```diff
@@ -1124,6 +1123,10 @@ func (r *OpenStackVMSetReconciler) createNewHostnames(
 			return newHostnames, err
 		}
 
+		if instance.Status.VMHosts == nil {
+			instance.Status.VMHosts = map[string]ospdirectorv1beta1.HostStatus{}
+		}
```

Comment:
Same question here regarding this initialization as with BaremetalSet controller

Reply:
same